garage/tf/regressors/gaussian_mlp_regressor_model.py

```diff
@@ -0,0 +1,98 @@
+"""GaussianMLPRegressorModel."""
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.models import GaussianMLPModel
+
+
+class GaussianMLPRegressorModel(GaussianMLPModel):
```

Comment:
Should this class in model module or regressor? @ryanjulian

Reply:
It was in model module. Ryan said it is regressor-specific and better to put it in regressor module.

Reply:
I c. Thanks for pointing it out. I’m making this resolved.